examples/leaderboard/client/PlayerList.jsx

```diff
@@ -1,30 +1,29 @@
+var PropTypes = require('prop-types');
 var React = require('react');
 var Player = require('./Player.jsx');
 var _ = require('underscore');
 
-var PlayerList = React.createClass({
-  propTypes: {
-    players: React.PropTypes.array.isRequired,
-    selectedPlayerId: React.PropTypes.string
-  },
+function PlayerList(props) {
```

Comment:
Did you intentionally leave this as a function to serve as an example of ES5 syntax, instead of converting it to a `class` like the examples higher up?

Though if it's to serve as an ES5 example, you can't use object destructuring (line 7) and spread syntax (line 14, though that might be ok if React has its own JS expression parser for template strings).

Reply:
Using just a function is how you write "stateless components" in React. In the docs they refer to it as function and class components: https://reactjs.org/docs/components-and-props.html

Here's another link about it: https://dev.to/mcarpenter/react---stateless-vs-stateful-components--5c3m